src/main/java/edu/cornell/kfs/sys/batch/service/DigestorXMLBatchInputFileType.java

```diff
@@ -0,0 +1,89 @@
+package edu.cornell.kfs.sys.batch.service;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+
+import org.apache.commons.digester3.Digester;
+import org.apache.commons.digester3.binder.DigesterLoader;
+import org.apache.commons.digester3.binder.RulesModule;
+import org.apache.commons.digester3.xmlrules.FromXmlRulesModule;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.net.URL;
+import javax.xml.validation.Schema;
+
+import org.kuali.kfs.sys.batch.XmlBatchInputFileTypeBase;
+import org.kuali.kfs.sys.exception.ParseException;
+import org.kuali.kfs.sys.exception.XmlErrorHandler;
+
+// CU customization to continue support for commons-digester after KualiCo has removed it from base code;
+// This can be removed once all code has been updated to use jaxb or other framework to replace commons-digester.
+public abstract class DigestorXMLBatchInputFileType extends XmlBatchInputFileTypeBase {
```

Comment:
Would it be better to name this class `DigesterXmlBatchInputFileType` instead? (Note the changes in spelling and uppercasing.) I know some of the fields and arguments were using the "digestor" naming in base code, but I would recommend using the "digester" name for the class itself, for consistency with the Digester3 library that it uses.

Also, KualiCo updated `XmlBatchInputFileTypeBase` to use generics for specifying the parsed object type. I would suggest updating this class to either add generics usage or to have it extend `XmlBatchInputFileTypeBase<Object>`.

Reply:
updated